zilliqa/src/api/types/eth.rs

```diff
@@ -24,6 +24,63 @@ pub enum HashOrTransaction {
     Transaction(Transaction),
 }
 
+#[derive(Clone, Serialize)]
+pub struct QuorumCertificate {
+    #[serde(serialize_with = "hex")]
+    pub signature: Vec<u8>,
+    pub cosigned: String,
+    #[serde(serialize_with = "hex")]
+    pub view: u64,
+    #[serde(serialize_with = "hex")]
+    pub block_hash: B256,
+}
+
+impl QuorumCertificate {
+    pub fn from_qc(qc: &message::QuorumCertificate) -> Self {
+        Self {
+            signature: qc.signature.to_bytes(),
+            cosigned: qc
```

Comment:
This isn't all that relevant, but I'll just leave here that not listing the committee in this structure makes me slightly nervous. You can get it easily enough from `eth_call:getStakers()` with block = block_hash, but doing these side-calls when you're writing bridging code has been gnarly for me in the past, and therefore this data structure gives me an undefined sense of the heebie-jeebies.

Tagging @JamesHinshelwood in case he can spot anything more concrete.

Reply:
Nothing stops us from adding new fields later so lets resolve this for now, see how good/bad the client UX is and add more if needed.